Week2/homework/maartjes_work.js

```diff
@@ -43,5 +43,18 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
+console.log(tasks);
+
+const fromMinToHours = tasks.map(tasks => tasks.duration / 60);
+console.log(fromMinToHours);
+
+const underTwoHours = tasks.filter(tasks => tasks.duration > 120);
+console.log(underTwoHours);
+
+const maartjeSalary = tasks.map((tasks) => tasks.duration * 20);
+console.log(maartjeSalary);
+
+const amount = tasks.reduce((maartjeSalary, tasks) => maartjeSalary + tasks.duration, 0);
```

Comment:
try a bit more descriptive in naming your variables/functions. This is called amount, but you also have a functioned called maartjeSalary. I think 'amount'  is basically also calculating the salary of maartje, so why do you name this 'amount' and the other 'maartjeSalary'.

Instead, you could think of naming it something like:  salaryPerTask and the other totalSalary